<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BTSRStartStudy</class>
 <widget class="QDialog" name="BTSRStartStudy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(41, 188, 204);
color:white;
font-size:28px;
}
QToolButton{
background-color:transparent;
color:white;
font-size:24px;
border:1px solid white;
}
QLabel{
color:white;
font-size:28px;
}</string>
  </property>
  <widget class="QPushButton" name="return_btn">
   <property name="geometry">
    <rect>
     <x>752</x>
     <y>380</y>
     <width>48</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10;background-color: rgb(100, 100, 100);font-size:22px;color:rgb(255, 255, 0);

</string>
   </property>
   <property name="text">
    <string>返
回</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>开始学习</string>
   </property>
   <widget class="QPushButton" name="pushButton_study">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10;background-color: rgb(100, 100, 100);font-size:22px;color:rgb(255, 255, 0);

</string>
    </property>
    <property name="text">
     <string>开始学习</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow;</string>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>学习测试</string>
   </property>
   <widget class="QPushButton" name="pushButton_startplustest">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10;background-color: rgb(100, 100, 100);font-size:22px;color:rgb(255, 255, 0);

</string>
    </property>
    <property name="text">
     <string>开始脉冲测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plusstudytest">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10;background-color: rgb(100, 100, 100);font-size:22px;color:rgb(255, 255, 0);

</string>
    </property>
    <property name="text">
     <string>脉冲学习测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopplustest">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10;background-color: rgb(100, 100, 100);font-size:22px;color:rgb(255, 255, 0);

</string>
    </property>
    <property name="text">
     <string>结束脉冲测试</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow;</string>
    </property>
    <property name="text">
     <string>状态</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="return_autotest">
   <property name="geometry">
    <rect>
     <x>752</x>
     <y>280</y>
     <width>48</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10;background-color: rgb(100, 100, 100);font-size:18px;color:rgb(255, 255, 0);}
QPushButton::checked{
color:red;}

</string>
   </property>
   <property name="text">
    <string>自
动
测
试</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
